Docker build failing because of new node image update from alpine 3.16 -> 3.17

https://hub.docker.com/layers/library/node/19-alpine/images/sha256-3f3816755f0ca539b3967e3642e1905fb804968fbba7c3e1c425ca08b619b086?context=explore

https://github.com/nodejs/docker-node/commit/aee40d3fd78f2053350d1480b792ca8119040a92

```
#0 3.093 src/commands/credits/modules/top/index.ts(1,10): error TS2305: Module '"@prisma/client"' has no exported member 'GuildMember'.
#0 3.094 src/handlers/database/index.ts(6,20): error TS7006: Parameter 'params' implicitly has an 'any' type.
#0 3.094 src/handlers/database/index.ts(6,28): error TS7006: Parameter 'next' implicitly has an 'any' type.
#0 3.094 src/helpers/credits/give.ts(6,43): error TS7006: Parameter 'tx' implicitly has an 'any' type.
#0 3.094 src/helpers/credits/set.ts(6,43): error TS7006: Parameter 'tx' implicitly has an 'any' type.
#0 3.094 src/helpers/credits/take.ts(6,43): error TS7006: Parameter 'tx' implicitly has an 'any' type.
#0 3.094 src/helpers/credits/transfer.ts(6,43): error TS7006: Parameter 'tx' implicitly has an 'any' type.
#0 3.094 src/schedules/shop/modules/roles/components/dueForPayment.ts(4,10): error TS2305: Module '"@prisma/client"' has no exported member 'GuildShopRoles'.
#0 3.094 src/schedules/shop/modules/roles/components/overDueForPayment.ts(4,10): error TS2305: Module '"@prisma/client"' has no exported member 'GuildShopRoles'.
```

There are two choices, either fix the issue so that prisma works with alpine 3.17 or just pin it to 3.16

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM node:19-alpine AS builder
+FROM node:19-alpine3.16 AS builder
 
 # Create app directory
 WORKDIR /app
@@ -15,7 +15,7 @@ COPY . .
 
 RUN npm run build
 
-FROM node:19-alpine
+FROM node:19-alpine3.16
 
 COPY --from=builder /app/node_modules ./node_modules
 COPY --from=builder /app/package*.json ./

